### PR TITLE
CSI-5283 change retention days to 90

### PR DESCRIPTION
**Pull request by samishekh**

### CODE REVIEW

File: .github/workflows/volume_group_operator_ci.yaml [hunk 1]
```diff
@@ -96,4 +96,4 @@ jobs:
         with:
           name: images
           path: /tmp/images.txt
-          retention-days: 7
+          retention-days: 300
```
Comment (matancarmeli7) [inline .github/workflows/volume_group_operator_ci.yaml:99]: why 300? IMO 100 is enough

Comment (samishekh) [reply .github/workflows/volume_group_operator_ci.yaml]: Barak suggested that I do it for 300 days if it is possible and does not affect anything.
but I got a warning in the build that I can do at max for 90 days. so I'm changed it.

**Review by matancarmeli7 — APPROVED** [review]
LGTM

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Pull request merged**